infrastructure/lib/addons/fluxv2.ts

```diff
@@ -10,10 +10,12 @@ export interface FluxV2Props extends StackProps {
   repoUrl: string;
   repoBranch: string;
   repoPath: string;
+  repoAddonPaths?: string[];
   testRepoName: string;
   testRepoUrl?: string;
   testRepoBranch?: string;
   testRepoPath?: string;
+  testRepoAddonPaths?: string[];
```

Comment:
why would you  need two arrays like one [here](https://github.com/awslabs/kubernetes-iteration-toolkit/pull/306/files#diff-bbf436cf80554d1c16a136a08cbc43d87d488d7ab9d51b6ca1aa66a74de555b9R13) and on this line ? Can't we just have one.

Reply:
From my current understanding, we support `Repo` and `testRepo` (this is not a great name, but we are using it now), user may want to install the addons from base `Repo` and (or) `testRepo`. We should provide both options IMO.

Reply:
The idea there was to have `repo` for k8s addons that needs to be installed on Management cluster through Flux and `testRepo` is a place holder for tekton tasks tests etc that we wanted to separate out at that time from the Infrastructure code repository itself. 


Given with this PR, now we can take in array of repos, this logical separation of `repos` and `testRepos` may not be needed at this point. Because customers can bring in any number of arbitrary set of `testrepos` which contain tests to sync on cluster and/or also arbitrary set of k8s addons can be installed  `repos` using the same array list . Doesn't add any benefit now to have separate vars for this. Makes sense ?